### ISSUE TITLE
finding composite breaks when there are no extra layers

### ISSUE THREAD

**Issue by Xeverous** [cycfi/elements#98]
The following change breaks radio buttons in the button example:

```diff
diff --git a/examples/buttons/main.cpp b/examples/buttons/main.cpp
index c5d99de..b450000 100755
--- a/examples/buttons/main.cpp
+++ b/examples/buttons/main.cpp
@@ -95,9 +95,22 @@ auto make_controls(view& view_)
             margin({ 10, 10, 20, 20 },
                top_margin(25,
                   vtile(
-                     top_margin(10, align_left(radio_button1)),
-                     top_margin(10, align_left(radio_button2)),
-                     top_margin(10, align_left(radio_button3))
+                     radio_button1,
+                     radio_button2,
+                     radio_button3
                   )
                )
             )

```

**Comment by djowel** [comment]
It would be nice if you can also post a picture to make things visual.

**Comment by djowel** [comment]
Ah, I see. Good catch. Will fix asap.

**Comment by djowel** [comment]
fixed in develop. thanks for the catch!

**Comment by djowel** [comment]
Please close after you verify the behavior is correct.

**Comment by Xeverous** [comment]
Yes, the issue is fixed.

**Comment by djowel** [comment]
bad fix (https://github.com/cycfi/elements/issues/107). I'll need to revert this and find a better fix.

**Comment by djowel** [comment]
fixed again in develop: 88a0eedf73fbc680243bbda27b1ce20d47eda359
